composer.json

```diff
@@ -22,7 +22,7 @@
     "minimum-stability": "stable",
     "require": {
         "composer-plugin-api": "^2.6",
-        "php-tuf/php-tuf": "0.1.3",
+        "php-tuf/php-tuf": "^0.1.4",
```

Comment:
Should we pin this exactly? I think we ran into a problem before where we make a change to `php-tuf/php-tuf` that then breaks this plugin but don't have time to update this plugin. 

Also if we get people testing the workflow then if we need to fix/improve something in `php-tuf/php-tuf` that might break this plug-in as soon as we make a new tag of `php-tuf/php-tuf` it might break this plug-in and then anyone who is testing the workflow has a broke system.

Reply:
Obviously once we are at 1.0.0 of the plugin we won't do this but during the 0.1.x phase it might be better

Reply:
Makes sense to me. Done.